packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -143,11 +146,12 @@ class _PropertyInputState extends State<_PropertyInput> {
 
   @override
   Widget build(BuildContext context) {
+    final argument = widget.argument;
     final decoration = InputDecoration(
-      helperText: '',
-      errorText: widget.argument.errorText,
+      helperText: argument.isRequired ? '*required' : '',
+      errorText: argument.errorText,
       isDense: true,
-      label: Text(widget.argument.name),
+      label: Text('${argument.name}${argument.isRequired ? '* ' : ''}'),
```

Comment:
do we want a trailing space after the * char?

Reply:
good point, no we don't!